src/sql/workbench/contrib/notebook/browser/cellViews/code.css

```diff
@@ -108,3 +108,17 @@ code-component .hide-component-button {
 	background-position: center;
 	background-color: transparent;
 }
+
+code-component .parameter {
+	bottom: 0;
+	margin-right: 15px;
+	position: absolute;
+	right: 0;
+}
+
+code-component .parameter span {
+	background-color: #0078D4;
```

Comment:
You've verified this passes accessibility contrast ratio requirements in light, dark and high contrast?

Reply:
@VasuBhog It would be best to keep color declarations out of CSS and instead use the colorRegistry for element colors. 

Take a look here for an example of button background color use:
https://github.com/microsoft/azuredatastudio/blob/797c89b023867f310c20182d551bc15b40e920df/src/sql/workbench/contrib/notebook/browser/notebookStyles.ts#L25

This references the color from the registry here:
https://github.com/microsoft/azuredatastudio/blob/797c89b023867f310c20182d551bc15b40e920df/src/vs/platform/theme/common/colorRegistry.ts#L231